adyenofficial.php

```diff
@@ -466,7 +466,8 @@ private function removeConfigurationsFromDatabase()
             'ADYEN_APPLE_PAY_MERCHANT_NAME',
             'ADYEN_APPLE_PAY_MERCHANT_IDENTIFIER',
             'ADYEN_GOOGLE_PAY_GATEWAY_MERCHANT_ID',
-            'ADYEN_GOOGLE_PAY_MERCHANT_IDENTIFIER'
+            'ADYEN_GOOGLE_PAY_MERCHANT_IDENTIFIER',
+            'ADYEN_REMOVE_PAYMENT_DISPLAY_COLLAPSE'
```

Comment:
because of the release strategy it would be easier to release a next version if the configuration was the other way around. Instead of removing the display collapse if it was enabling it.

In this case we can keep the default value as false when we release a new feature version so we won't break anything and in a next major release we can change it to by default true.

```suggestion
            'ADYEN_ENABLE_PAYMENT_DISPLAY_COLLAPSE'
```

Reply:
Change committed